CHANGELOG/v1.6.0.md

```diff
@@ -0,0 +1,294 @@
+- CI: Fix .golangci.yml comments (#9499)
+- CI: Ginkgolinter: forbid focus container (#9320)
+- CI: Github: add edited and reopened as triggers for the GH workflow approval (#9259)
+- CI: Github: add workflow to auto-approve golangci-lint if ok-to-test label is set (#9244)
+- CI: Go.mod reformat to have only two require blocks (#9192)
+- CI: Golangci-lint: replace deprecated local-prefixes setting for gci (#9339)
+- CI: Make GO_ARCH explicit in verify_containter script (#9341)
+- CI: Update actions for 1.5 and make names consistent (#9115)
+- CI: Verify plantuml image generation in CI (#9363)
+- ClusterCacheTracker: Add separate concurrency flag for cluster cache tracker (#9116)
+- ClusterCacheTracker: Ensure Get/List calls are not getting stuck when apiserver is unreachable (#9028)
+- ClusterCacheTracker: Fix accessor deletion on health check failure (#9025)
+- ClusterClass: Add additional test cases for nested variable defaulting (#9728)
+- ClusterClass: Add ownerRefs to BootstrapConfig/InfraMachinePool in classy Clusters (#9389)
+- ClusterClass: Embed ssa.FilterObjectInput into HelperOption to remove duplication (#9512)
+- ClusterClass: Fix some nits in Cluster topology engine tests (#9464)
+- ClusterClass: Improve Cluster variable defaulting/validation errors (#9452)
+- ClusterClass: Improve message for TopologyReconciledCondition (#9400)
+- ClusterClass: Make ClusterClass generated object names consistent (#9254)
+- ClusterClass: Minor fixes for CC+MP implementation (#9318)
+- clusterctl: Check resource blocking clusterctl move during discovery (#9246)
+- clusterctl: Propagate ctx to retryWithExponentialBackoff in clusterctl (#9437)
+- clusterctl: Use goproxy to check version in clusterctl (#9237)
+- clusterctl: Use http get to download files from GitHub in clusterctl (#9236)
+- Dependency: Bump cert-manager to v1.13.2 (#9653)
+- Dependency: Bump cloud.google.com/go/storage from 1.34.1 to 1.35.1 in /hack/tools (#9727)
+- Dependency: Bump controller tools to v1.13.0 (#9221)
+- Dependency: Bump controller-runtime to v0.16.3 (#9592)
+- Dependency: Bump conversion-gen to v0.28 (#9267)
+- Dependency: Bump corefile-migration library to v1.0.21 (#9307)
+- Dependency: Bump docker to v24.0.5 (#9064)
+- Dependency: Bump envtest binaries to 1.28 (#9268)
+- Dependency: Bump github.com/blang/semver to v4 (#9189)
+- Dependency: Bump github.com/docker/distribution (#9544)
+- Dependency: Bump github.com/docker/docker from 24.0.6+incompatible to 24.0.7+incompatible in /test (#9652)
+- Dependency: Bump github.com/emicklei/go-restful/v3 from 3.10.2 to 3.11.0 in /test (#9272)
+- Dependency: Bump github.com/evanphx/json-patch/v5 from 5.6.0 to 5.7.0 (#9397)
+- Dependency: Bump github.com/fatih/color from 1.15.0 to 1.16.0 (#9681)
+- Dependency: Bump github.com/go-logr/logr from 1.2.4 to 1.3.0 (#9644)
+- Dependency: Bump github.com/google/go-cmp from 0.5.9 to 0.6.0 (#9562)
+- Dependency: Bump github.com/onsi/ginkgo/v2 from 2.13.0 to 2.13.1 (#9716)
```

Comment:
Q: Could it be that we only have one changelog entry for dependencies that have been bumped multiple times?

E.g. ginkgo in release-1.5 was not v2.13.0 as it seems here in the changelog

Reply:
yes, we do remove "duplicated" entries for the same module
It's documented in the release process
We can discuss a process change, I don't mind having multiple entries for the same dependency